ledfx/integrations/mqtt_hass.py

```
@@ -558,18 +562,122 @@
+                    # )
+
+                # Handle effect selection
+                selected_effect_or_preset = payload.get("effect")
+                if selected_effect_or_preset:
+                    if selected_effect_or_preset == "back":
+                        effect_list = list(
+                            self._ledfx.effects.classes().keys()
+                        )
+                    elif (
+                        selected_effect_or_preset
+                        in self._ledfx.effects.classes().keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)